Orange/util.py

```diff
@@ -246,6 +248,13 @@ def __repr__(self):
                                  ', '.join('{}={!r}'.format(*pair)
                                            for pair in names_values))
 
+    @staticmethod
+    def __equal(obj1, obj2):
+        try:
+            return obj1.size == obj2.size and np.all(obj1 == obj2)
+        except AttributeError:
+            return np.all(obj1 == obj2)
```

Comment:
Or I could construct complex objects and use the designated `__repr__()` interface. I'm happy to do neither here. The numpy case is covered and tested. It should work until it won't and we'll figure it out then. :stuck_out_tongue_closed_eyes: